openapi_python_client/parser/properties/schemas.py

```diff
@@ -14,10 +18,66 @@
     ModelProperty = "ModelProperty"
 
 
+_ReferencePath = NewType("_ReferencePath", str)
+_ClassName = NewType("_ClassName", str)
+
+
+def parse_reference_path(ref_path_raw: str) -> Union[_ReferencePath, ParseError]:
+    parsed = urlparse(ref_path_raw)
+    if parsed.scheme or parsed.path:
+        return ParseError(detail=f"Remote references such as {ref_path_raw} are not supported yet.")
+    return cast(_ReferencePath, parsed.fragment)
+
+
+@attr.s(auto_attribs=True, frozen=True)
+class Class:
+    """ Info about a generated class which will be in models """
```

Comment:
I know what you mean here, but this docstring is a little confusingly worded

Reply:
Do you have a suggestion? Maybe "Info about a class to be generated"?

Reply:
Maybe "Represents Python class which will be generated from OpenAPI schema"?